### DISCUSSION TITLE
How can I pass --solution-folder to pac solution sync command?

### DISCUSSION THREAD

**Discussion by Zerajima** [microsoft/powerplatform-build-tools#772] [Q&A] [answered]
I can not figure out how to pass correct path to pac solution sync command. I have tried both relative and absolute paths, with no success. Does anybody have an example of how to use this parameter?

Given the folder structure:
- c:\_dev\project - root folder of project
- c:\_dev\project\solution - folder with dataverse solution and .cdsproj file

Here are some of the approaches I have tried (starting from c:\_dev\project"):
- pac solution sync --solution-folder solution: cli crashes
- pac solution sync --solution-folder .\solution: cli crashes
- pac solution sync --solution-folder \solution: cli crashes
- pac solution sync --solution-folder ./solution: cli crashes
- pac solution sync --solution-folder /solution: cli crashes
- pac solution sync --solution-folder c:\_dev\project\solution: cli crashes
- pac solution sync --solution-folder "c:\_dev\project\solution": cli crashes
- pac solution sync --solution-folder 'c:\_dev\project\solution': Error: The value passed to '--solution-folder' is invalid

The command works correctly without this parameter, if I run it from folder which contains .cdsproj file.

**Comment by tehcrashxor** [comment, marked as answer]
Working on a fix now.  
In the meantime, the current behavior is 
* search for `[--solution-folder]\Other\Solution.xml`
* use the .cdsproj in the current working directory

Note that the folder containing the .cdsproj does not contain `Other\Solution.xml`, but rather `src\Other\Solution.xml`, so until a fix is out you would need to either
1. `cd c:_dev\project\solution` then `pac solution sync`, or
2. `pac solution sync "c:_dev\project\solution\src"`

**Reply by Zerajima** [reply]
Thanks for help.
